recipes/doublemetaphone/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "DoubleMetaphone" %}
+{% set version = "0.1" %}
+{% set md5 = "86e5ae4a4839321f60b9801837eb01c1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win and linux]
```

Comment:
Hmm...says [POSIX]( https://github.com/datamade/doublemetaphone/blob/7688cb90b5fd1427f788cc0672183ccea94fc45d/setup.py#L19 ) upstream. Why would it not work on Linux then?